django/conf/global_settings.py

```diff
@@ -356,41 +356,43 @@ def gettext_noop(s):
 # https://docs.python.org/library/datetime.html#strftime-behavior
 # * Note that these format strings are different from the ones to display dates
 DATE_INPUT_FORMATS = [
-    '%Y-%m-%d', '%m/%d/%Y', '%m/%d/%y',  # '2006-10-25', '10/25/2006', '10/25/06'
-    '%b %d %Y', '%b %d, %Y',             # 'Oct 25 2006', 'Oct 25, 2006'
-    '%d %b %Y', '%d %b, %Y',             # '25 Oct 2006', '25 Oct, 2006'
-    '%B %d %Y', '%B %d, %Y',             # 'October 25 2006', 'October 25, 2006'
-    '%d %B %Y', '%d %B, %Y',             # '25 October 2006', '25 October, 2006'
+    '%Y-%m-%d',
+    '%m/%d/%Y',
+    '%m/%d/%y',  # '2006-10-25', '10/25/2006', '10/25/06'
+    '%b %d %Y',
+    '%b %d, %Y',  # 'Oct 25 2006', 'Oct 25, 2006'
+    '%d %b %Y',
+    '%d %b, %Y',  # '25 Oct 2006', '25 Oct, 2006'
+    '%B %d %Y',
+    '%B %d, %Y',  # 'October 25 2006', 'October 25, 2006'
+    '%d %B %Y',
+    '%d %B, %Y',  # '25 October 2006', '25 October, 2006'
```

Comment:
We might want to move these comments.

Reply:
Yeah.. If we wanted to, Black also supports:

```
# fmt: off
DATE_INPUT_FORMATS = [
 ...
]
# fmt: on
```